docs/using/staking.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Staking
+authors: "Tim McMackin"
+last_update:
+  date: 20 December 2024
+---
+
+Staking is the process of temporarily locking tez on the Tezos platform in exchange for rewards.
+Staked tez stays in the staker's account, but the staker cannot transfer (or spend) that tez.
+Stakers can unstake the tez at any time, which makes them spendable again after a delay.
+Staking is an important part of running the Tezos protocol and keeping the blockchain secure, so this is why stakers earn rewards proportional to the locked funds.
+
+Two main groups stake on Tezos:
+
+- Bakers, the creators of new blocks in Tezos, must stake 6,000 tez to receive the right to "bake" blocks and to receive rewards for baking those blocks.
+Their staked tez ensures that they bake correctly, because part of their stake is taken ("slashed") if they misbehave.
+For more information about staking for baking purposes, see [Baking](/architecture/bakers).
+
+- Any Tezos user can stake tez with a baker and earn rewards for them.
+In exchange for staking tez with a baker, users automatically receive a portion of the baker's rewards in proportion to how much they stake.
+Users can stake any amount of tez, but there is a limit to how much staked tez a single baker can accept, and bakers must opt in to allow users to stake with them.
+
+Staking has some similarities to the concept of a Smart Rollup bond, but there are important differences; see [Bonds](/architecture/smart-rollups#bonds).
+
+## Staking as a Tezos user
+
+The process of user staking has these main steps:
+
+1. The user selects a baker as a delegate for their account.
+As described in [Delegating to a baker](/architecture/bakers#delegating-to-a-baker), delegating tez to a baker puts that tez toward the baker's voting and baking power.
+Delegating incurs no risk to the user; the user retains full control of the tez, they can spend it at any time, and they are not punished if the baker misbehaves.
+
+1. The user stakes tez with the baker.
+Staking the tez locks it temporarily, but the tez stays in the user's account.
```

Comment:
(it's really a question, I just want to learn:)
so ok, "tez" can be singular or plural, but in this case it *is* plural, so don't we have to agree in number?
```suggestion
Staking the tez locks them temporarily, but the tez stay in the user's account.
```